src/protocols/strategies/convex.strategy.ts

```diff
@@ -269,24 +268,34 @@ export async function resolveCurvePoolTokenPrice(chain: Chain, token: Token): Pr
 }
 
 // TODO: this function is a bit weird, we can't assume to ever have a 'Harvest'
-async function retrieveBribesProcessorData(chain: Chain, vault: VaultDefinition): Promise<CachedValueSource[]> {
+async function retrieveHarvestForwarderData(chain: Chain, vault: VaultDefinition): Promise<CachedValueSource[]> {
   const sdk = await chain.getSdk();
-  const bribeProcessor = BribesProcessor__factory.connect('0xbed8f323456578981952e33bbfbe80d23289246b', sdk.provider);
+  const harvestForwarder = HarvestDistributor__factory.connect(
+    '0xA84B663837D94ec41B0f99903f37e1d69af9Ed3E',
```

Comment:
It looks like you only have a generic `tokens` config file.  I'd like to see no contract addresses in files as any changes to them can get easily lost and they're high security updates